nipap/nipap/smart_parsing.py

```diff
@@ -429,8 +433,6 @@ def _string_to_dictsql(self, part):
                     'val2': part[0]['word']
                     }
 
-        elif part.getName() == 'ipv4_address':
-            dictsql = {}
```

Comment:
There is actually an ipv4_address matching defined at https://github.com/SpriteLink/NIPAP/blob/master/nipap/nipap/smart_parsing.py#L103

but it's not in use. You can see how all other things are brought together in the parsing tree here https://github.com/SpriteLink/NIPAP/blob/master/nipap/nipap/smart_parsing.py#L117-L132 and that's where this PR adds ipv6_address & ipv6_prefix but there is no ipv4_addres. Things that are IPv4 addresses will be matched by the 'word' type.